devsetup/scripts/standalone.sh

```diff
@@ -117,7 +118,7 @@ export NTP_SERVER=${NTP_SERVER:-"clock.corp.redhat.com"}
 export EDPM_COMPUTE_CEPH_ENABLED=${EDPM_COMPUTE_CEPH_ENABLED:-true}
 export CEPH_ARGS="${CEPH_ARGS:--e \$HOME/deployed_ceph.yaml -e /usr/share/openstack-tripleo-heat-templates/environments/cephadm/cephadm-rbd-only.yaml -e \$HOME/nova_noceph.yaml}"
 export COMPUTE_DRIVER=${COMPUTE_DRIVER:-"libvirt"}
-export IP=${IP}
+export IP=${CTLPLANE_IP}
```

Comment:
I find the flow of values somewhat confusing... First we assign CTLPLANE_IP=IP, then we assign IP=CTLPLANE_IP... Could we just define a new variable ADDITIONAL_CTLPLANE_IP and below in the ansible variables have

```
additional_ctlplane_ip: ${ADDITIONAL_CTLPLANE_IP}
```
?

Reply:
And the `{% if additional_ctlplane_ip is defined %}` could perhaps become `{% if additional_ctlplane_ip is defined and additional_ctlplane_ip %}`? I'm not sure if it's necessary but it also rules out when the additional IP variable is defined but `null` or empty string.